### PR TITLE
model_part_io changes for bool mdpa

### PR DESCRIPTION
**Pull request by farrufat-cimne**
Line 800. The rest are only trailing whitespaces.
With this change, the mdpa has to be written as 'true' or 'false' besides the name of the bool VARIABLE,

### CODE REVIEW

File: kratos/sources/model_part_io.cpp [hunk 1]
```diff
@@ -797,7 +797,9 @@ namespace Kratos
                 bool temp;
 
                 ReadWord(value); // reading value
-                ExtractValue(value,temp);
+                std::stringstream str_stream(value);
```
Comment (RiccardoRossi) [inline kratos/sources/model_part_io.cpp]: what are u doing here? why is this needed?

in any case please add a test for the new feature, ideally it should fail before your PR

**Review by RiccardoRossi — DISMISSED** [review]
i am blocking principally since this is a file that affects everyone.

File: kratos/sources/model_part_io.cpp (hunk 1 above)
Comment (maceligueta) [reply kratos/sources/model_part_io.cpp]: ok, we'll put the test. @farrufat-cimne , any help is welcome.

Comment (philbucher) [reply kratos/sources/model_part_io.cpp]: Do you want to read booleans for variable-data?

If I am not completely wrong then I think I used this before
I think with `1` and `0` though

Comment (maceligueta) [reply kratos/sources/model_part_io.cpp]: @philbucher , according to @farrufat-cimne , we had problems at reading 0 and 1 yesterday, but we will make sure by writing the test proposed by @RiccardoRossi

Comment (RiccardoRossi) [reply kratos/sources/model_part_io.cpp]: most importantly, the ExtractValue function should be doing exactly what you added there.

did u try to use 

      ExtractValue<bool>(....)

Comment (maceligueta) [reply kratos/sources/model_part_io.cpp]: Not exactly, there's an extra order added. Let's check the test first, because maybe this dicsussion is useless. Need more time.

Comment (maceligueta) [reply kratos/sources/model_part_io.cpp]: @philbucher  was right and it works perfectly with 1 and 0.
I just pushed the test, and undid the previous changes.

**Comment by maceligueta** [comment]
So this PR got reduced to just the addition of some verifications inside the test_model_part_io.
No changes in the code.
Can it be approved?

**Review by philbucher — APPROVED** [review]
Thx for adding the test :+1:

**Comment by philbucher** [comment]
Ping @RiccardoRossi

**Comment by maceligueta** [comment]
Ping @RiccardoRossi
